src/widget/paintable.cpp

```diff
@@ -134,6 +134,10 @@ QRectF Paintable::rect() const {
     return QRectF();
 }
 
+QImage Paintable::toImage() const {
+    return m_pPixmap.isNull() ? QImage() : m_pPixmap->toImage();
```

Comment:
a null QPixmap isn't the same as a null QImage. so the conditonal says: if m_pPixmap is a null QPixmap, return a null QImage, otherwise return the m_pPixmap converted to QImage. makes sense?

Reply:
If something is not obvious then please add comments to the code instead of answering questions here in the PR were the answer is lost after merging.

Ideally, code doesn't need to be commented and is self-explaining. Inline code comments should be considered a crutch.

Reply:
Right, but won't a null QPixmap converted to a QImage result in a null QImage?

Reply:
No, but the qt docs are indeed very clear. I will remove the ternary expression.